bentoml/cli/aws_ec2.py

```diff
@@ -0,0 +1,308 @@
+    @click.option(
+        "-n",
+        "--namespace",
+        type=click.STRING,
+        help='Deployment namespace managed by BentoML, default value is "dev" which '
+        'can be changed in BentoML configuration yatai_service/default_namespace',
+    )
+    @click.option(  # pylint: disable=unused-variable
+        "-o", "--output", type=click.Choice(["json", "yaml", "table"]), default="json"
+    )
+    def get(name, namespace, output):
+        yatai_client = get_default_yatai_client()
+        describe_result = yatai_client.deployment.describe(namespace, name)
+
+        if describe_result.status.status_code != yatai_proto.status_pb2.Status.OK:
+            error_code, error_message = status_pb_to_error_code_and_message(
+                describe_result.status
+            )
+            raise CLIException(f"{error_code}:{error_message}")
+
+        get_result = yatai_client.deployment.get(namespace, name)
+        if get_result.status.status_code != yatai_proto.status_pb2.Status.OK:
+            error_code, error_message = status_pb_to_error_code_and_message(
+                describe_result.status
+            )
+            raise CLIException(f"{error_code}:{error_message}")
+
+        _print_deployment_info(get_result.deployment, output)
+
+    @aws_ec2.command(help="Update existing AWS EC2 deployments")
+    @click.argument("name", type=click.STRING)
+    @click.option(
+        "-b",
+        "--bento",
+        "--bento-service-bundle",
+        type=click.STRING,
+        callback=parse_bento_tag_callback,
+        help="Target BentoService to be deployed, referenced by its name and version "
+        'in format of name:version. For example: "iris_classifier:v1.2.0"',
+    )
+    @click.option(
+        "-n",
+        "--namespace",
+        type=click.STRING,
+        help='Deployment namespace managed by BentoML, default value is "dev" which '
+        'can be changed in BentoML configuration yatai_service/default_namespace',
+    )
+    @click.option(
+        "-o", "--output", type=click.Choice(["json", "yaml", "table"]), default="json"
+    )  # pylint: disable=unused-variable
+    @click.option(
+        "--wait/--no-wait",
+        default=True,
+        help="Wait for apply action to complete or encounter an error."
+        "If set to no-wait, CLI will return immediately. The default value is wait",
+    )
+    def update(name, namespace, bento, output, wait):
+        yatai_client = get_default_yatai_client()
+        if bento:
+            bento_name, bento_version = bento.split(":")
+        else:
+            bento_name = None
+            bento_version = None
+
+        with Spinner("Updating EC2 deployment"):
+            update_result = yatai_client.deployment.update_ec2_deployment(
+                deployment_name=name,
+                bento_name=bento_name,
+                bento_version=bento_version,
+                namespace=namespace,
+                wait=wait,
```

Comment:
Should we provide the same configuration like min-capacity, instance-type for update as well?

Also, what would happen, if we don't provide `bento`?

Reply:
If bento not provided,its taken from existing deployment.
I have done this with respect to existing format of other deployments like lambda.

Reply:
Added similar options for update